client/src/pages/Landing.tsx

```diff
@@ -1,27 +1,195 @@
+    //     </div>
+    //   </div>
+    // </>
     <>
-      <main>
-        <h2 className="font-bold mt-10 text-emerald-500">
-          This is the landing page
-        </h2>
-        <span className="material-symbols-outlined">expand_more</span>
-      </main>
-
-      <Button variant="filled">Filled</Button>
-      <Button variant="filled" disabled><p>Filled</p> + <p>Disabled</p></Button>
-      <Button variant="tonal">Tonal</Button>
-      <Button variant="tonal" disabled>Tonal + Disabled</Button>
-      <Button variant="text" onClick={() => alert(111)}>Text</Button>
-      <Button variant="text" disabled>Text + Disabled</Button>
-
-      <nav>
-        <Link to="/upload">Get Started</Link>
-        <Link to="/workflow">Learn more</Link>
-      </nav>
+      <LogoBar/>
+      <div className="flex flex-row px-28 mt-28">
+        <div className="w-7/12 m-0 pl-12">
+          <div className="text-5xl font-poppins font-[800] leading- tracking-tight">
+            <h1>A better way to</h1>
+            <h1 className=" text-primary-50">organize deadlines</h1>
+          </div>
+          <div className="leading-relaxed font-poppins mt-10 w-11/12">
+            <p className="text-xl font-normal non-italic">
+              Lifeline takes the stress out of managing important dates for your
+              courses. Start saving more time and always keep track of your
+              deadlines in just a few clicks.
+              <span className="block">
+                Start for free today to make your scheduling a breeze.
+              </span>
+            </p>
+            <nav>
+              <div className="mt-[60px]">
+                <Link
+                  to="/upload"
+                >
+                  <Button
+                    onClick={routeChange}
+                    variant="filled"
+                  > 
+                      Get Started 
+                  </Button>
+                </Link>
+                <Link
+                  to="/workflow"
+                  className="text-learnMore text-sm text-semibold py-3.5 px-6 inline"
+                >
+                  Learn more
+                </Link>
+              </div>
+            </nav>
+          </div>
+        </div>
+
+        <div className="w-5/12">
+          <img src={illustration1} alt="illustration 1" />
+        </div>
+      </div>
+
+      <div className="flex flex-row w-4/5 mt-36 mx-auto font-poppins">
+        <div className="w-1/2">
+          <img src={illustration19} alt="illustration 19" />
+        </div>
+        <div className="pt-8 w-1/2 pl-16">
+          <div className="w-4/5 h-1/3">
+            <div className="flex flex-row">
+              <img src={uploadIcon} alt="upload" />
+              <div className="ml-4">
+                <p className="text-lg block font-[600]">
+                  Upload your course outline
+                  <span className="block text-sm font-[400]">
+                    Up to 3 PDF documents are accepted
```

Comment:
Minor verbiage thing here. As I mentioned in the prototype: 
> It makes no sense when most of people have (4 or) 5 courses in one semester and only 3 files could be uploaded at once, bad user experience; and we don't even know a specific limit, so leaving the verbiage here as it is.

Maybe do 5 or 6, or don't use a limit since we haven't figured out.